src/main/java/com/cinema/Application.java

```diff
@@ -66,11 +68,11 @@ public static void main(String[] args) throws AuthenticationException {
 
         authenticationService.register(roma.getEmail(), roma.getPassword());
         User romaFromDB = authenticationService.login(roma.getEmail(), roma.getPassword());
```

Comment:
let's wrap call to `login()` method in try-catch and add log.warn with the informative message in `catch` block